src/components/Header/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import routes from '../../constants/routes.js';
+import RethinkLogo from '../../images/header/RethinkLogo.jpg';
+import './styles.css';
+
+// defuzz logo
+// buttons
+// clamp css for responsivity
+export default function Header() {
+  return (
+    <div>
+      <section className="logo">
+        <img src={RethinkLogo} alt="The word ReThink with petals next to it" />
```

Comment:
Let's change the alt text to just "ReThink Logo". Its a little too verbose for alt text